packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/__tests__/quanticSearchBoxInput.test.js

```diff
@@ -353,6 +353,33 @@ describe('c-quantic-search-box-input', () => {
               functionsMocks.exampleHandleSubmitSearch
             ).toHaveBeenCalledTimes(1);
           });
+
+          it(`should ${
+            textareaValue ? 'not ' : ''
+          }dispatch a #quantic__submitsearch custom event when the shift key is pressed`, async () => {
```

Comment:
wouldn't it be more like `shift+enter key is pressed`?

Reply:
Yes, pressing enter is mentioned the parent `describe` block of this test but we can make more simple yeah.